apis/v1alpha1/generator.yaml

```diff
@@ -54,12 +54,80 @@ resources:
         - InvalidParameterValue
         - MissingParameter
     fields:
+      EnableNetworkIsolation:
+        late_initialize:
+          min_backoff_seconds: 5
       Tags:
         compare:
           is_ignored: true
     hooks:
       delta_pre_compare:
         code: customSetDefaults(a, b)
+  InferenceComponent:
+    reconcile:
+      requeue_on_success_seconds: 30
+    update_conditions_custom_method_name: CustomUpdateConditions
+    exceptions:
+      errors:
+        404:
+          code: ValidationException
+          message_prefix: Could not find inference component
+      terminal_codes:
+        - InvalidParameterCombination
+        - InvalidParameterValue
+        - MissingParameter
+        # Custom error
+        - InferenceComponentUpdateError
+    hooks:
+      sdk_read_one_post_set_output:
+        template_path: inference_component/sdk_read_one_post_set_output.go.tpl
+      sdk_update_pre_build_request:
+        template_path: inference_component/sdk_update_pre_build_request.go.tpl
+      sdk_update_post_set_output:
+        template_path: inference_component/sdk_update_post_set_output.go.tpl
+      sdk_delete_pre_build_request:
+        template_path: common/sdk_delete_pre_build_request.go.tpl
+      sdk_delete_post_request:
+        template_path: common/sdk_delete_post_request.go.tpl
+    fields:
```

Comment:
I see that theres a CurrentCopyCount(https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sagemaker/client/describe_inference_component.html), does that need to be included here as well.